test-containers/karaf/karaf-run/src/test/java/com/vaadin/flow/karaf/smoketest/PushResourceIT.java

```diff
@@ -24,6 +24,11 @@
 
 public class PushResourceIT extends ChromeBrowserTest {
 
+    @Override
+    protected int getDeploymentPort() {
+        return 8181;
```

Comment:
`main` branch has a bit different code, why it differs for 7.1?

Reply:
> The error `java.lang.IllegalArgumentException: invalid flag: --release` looks as not related to the change. Let's merge and check the Bender build.

It is because the validation jobs always builds against flow 23.3, but for this PR it is running java 8 that does not support the `release` flag

Reply:
> `main` branch has a bit different code, why it differs for 7.1?

On `main` code has been changed to fix flaky tests (bbbc1c7ea2090e1e619ea0c19fd84cf9405619ce), but it seems there was no backport